src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -286,13 +320,18 @@ public String toString() { // TODO Use a library for this kind of tasks, maybe L
                 + ", flashbackDuration=" + flashbackDuration
                 + ", output=" + output
                 + ", fontSize=" + fontSize
+                + ", fontName=" + fontName
+                + ", fontStyle=" + fontStyle
                 + ", markerSize=" + markerSize
                 + ", waypointSize=" + waypointSize
                 + ", keepLastFrame=" + keepLastFrame
                 + ", logo=" + logo
+                + ", logoPosition=" + logoPosition
+                + ", attributionPosition=" + attributionPosition
+                + ", photoAnimationDuration=" + photoAnimationDuration
+                + ", informationPosition=" + informationPosition
                 + ", photoDirectory=" + photoDirectory
                 + ", photoTime=" + photoTime
-                + ", photoAnimationDuration=" + photoAnimationDuration
```

Comment:
You moved this line four lines up into the block of the position settings. I would prefer to have this line stay here together with the other photo related lines.